lib/archethic/mining/standalone_workflow.ex

```diff
@@ -341,4 +344,14 @@ defmodule Archethic.Mining.StandaloneWorkflow do
       |> P2P.broadcast_message(%NotifyPreviousChain{address: tx.address})
     end
   end
+
+  defp maybe_add_burning_address(resolved_addresses, burning_node, tx_mvts) do
+    case Enum.any?(tx_mvts, fn tx_mvt -> tx_mvt.to == burning_node end) do
+      true ->
+        Enum.concat(resolved_addresses, [LedgerOperations.burning_address()])
```

Comment:
same as @bchamagne  comment.
maybe we can move this into a module to avoid repeating code.